curriculum/challenges/english/16-the-odin-project/top-learn-css-foundations-projects/css-foundations-exercise-c.md

```diff
@@ -7,26 +7,100 @@ dashedName: css-foundations-exercise-c
 
 # --description--
 
-description
+Let's build a little off the previous exercise, in which you added multiple classes to a single element in order to apply two different rules to it.
+
+1. You should see a black background and white text on the first element.
+1. You should see a yellow background on the second element.
```

Comment:
```suggestion
1. You should see a black background and white text on the first `button` element.
1. You should see a yellow background on the second `button` element.
```

It's not really clear what elements should be styled. `head` and `body` are also elements.